core/src/main/java/org/kiwix/kiwixmobile/core/search/AutoCompleteAdapter.java

```diff
@@ -139,11 +150,11 @@ protected FilterResults performFiltering(CharSequence constraint) {
 
     @Override
     protected void publishResults(CharSequence contraint, FilterResults results) {
-      mData = (ArrayList<String>) results.values;
+      data = (ArrayList<String>) results.values;
       if (results.count > 0) {
         notifyDataSetChanged();
       } else {
-        notifyDataSetInvalidated();
+        //notifyDataSetInvalidated();
```

Comment:
it either needs to be here or it doesn't, leaving commented out code and an entire execution branch doesn't serve much of a purpose